### PR TITLE
feat: add WDIO v7 basic example

### PR DESCRIPTION
**Pull request by Zidious**
This PR adds a V7 WDIO example to hopefully assist customers that are using watcher and encounter type errors due to the removal of typings in v8 release.

ref: https://github.com/dequelabs/jazzband/issues/533

### CODE REVIEW

File: wdio/v7/typescript-basic/setup.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import 'mocha'
+import { wdioConfig, WdioController, wrapWdio } from '@axe-core/watcher'
+import { remote, type Browser } from 'webdriverio'
+
+/* Get your configuration from environment variables. */
+const { API_KEY, SERVER_URL = 'https://axe.deque.com' } = process.env
+
+/**
+ * As of V8, WebdriverIO removed the deprecated the @wdio/sync package. To use this on V7 of WebdriverIO
```
Comment (stephenmathieson) [inline wdio/v7/typescript-basic/setup.ts:9]:
```suggestion
 * As of V8, WebdriverIO removed the deprecated the @wdio/sync package. To use @axe-core/watcher on V7 of WebdriverIO
```

File: wdio/v7/typescript-basic/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "lib": ["es2016", "dom"],
+    "types": ["webdriverio/async"]
```
Comment (stephenmathieson) [inline wdio/v7/typescript-basic/tsconfig.json:4]: Why is this needed?

Comment (Zidious) [reply wdio/v7/typescript-basic/tsconfig.json:4]: This resolves a type error when viewing the types from WDIO v7
image


it doesn't affect the types within the tests but to resolve the type inspection, thought I'd add it: 

```
NOTE: If you hit "error TS2694: Namespace 'global.WebdriverIO' has no exported member 'Browser'" when using typescript,
 * add "webdriverio/async" into tsconfig.json's "types" array will solve it: <code> { "compilerOptions": { "types": ["webdriverio/async"] } } </code>
```

**Review by stephenmathieson — APPROVED** [review]
